build.gradle.kts

```diff
@@ -508,6 +508,7 @@ tasks {
     description = "Runs the integration tests."
     sharedIntegrationTestConfig(buildCodyDir, "replay")
     dependsOn("processIntegrationTestResources")
+    project.properties["repeatTests"]?.let { systemProperty("repeatTests", it) }
```

Comment:
If you need it to be a system property you don't need to do that, but instead you can run gradle with `-DrepeatTests` instead of `-PrepeatTests`

Reply:
According to [Stackoverflow](https://stackoverflow.com/a/21406600), the `-D` argument won't necessarily pass through to the tests.